server/app.js

```diff
@@ -141,6 +141,8 @@ app.use(
         "gist.github.com",
         "www.google-analytics.com",
         "browser.sentry-cdn.com",
+        "cdn.jsdelivr.net",
+        "storage.googleapis.com",
```

Comment:
Can we add this without adding two more third-party dependencies? I really don't want to add the entirety of these two CDN's to the CSP allow-list and I think it would be beneficial to keep dependencies bundled with the application.

Reply:
We certainly can host these in outline in public folder or using yarn. But if only you can guide me, where or how, it'll make my life easy :)